### PR TITLE
Created the ICBT folder and added details of the current committee

### PR DESCRIPTION
**Pull request by Poojith-Obeysekara**

### CODE REVIEW

File: cc/ICBT/2022-current.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## How to join?
+
+Any student studying within ICBT who’s interested in joining with us , can sign up and join the community as a volunteer.
+- Initially, fill out the Google Form by providing accurate details : https://forms.gle/EPQ4nK4ziW1M8ySQ8
+- Thereafter , you’ll be contacted by one of the Executive Committee Members and will be  added as a member to our WhatsApp Group.
+
+## MIC/Board information
+
+* Master In-Charge – Mr. Induranga De Silva. (mailto: induranga@icbtcampus.edu.lk)
```
Comment (sdglhm) [inline cc/ICBT/2022-current.md]: Change @ and . with (at) and (dot) to prevent basic scrapers from scraping the email.

**Review by sdglhm — COMMENTED** [review]
Please see the comment, other than that, LGTM

File: cc/ICBT/2022-current.md (hunk 1 above)
Comment (Poojith-Obeysekara) [reply cc/ICBT/2022-current.md]: Sure aiya.

Comment (sdglhm) [reply cc/ICBT/2022-current.md]: Thank you

**Review by sdglhm — APPROVED** [review]
LGTM

**Comment by sdglhm** [comment]
@dhanushkac Over to you

**Review by sdglhm — DISMISSED** [review]

File: cc/ICBT/2022-current.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# ICBT- International College of Business and Technology
+ 
+Our sole objective is to motivate students in our vibrant community to expand their knowledge in the latest tech industry and to assist them in contributing to open-source projects in the most effective and productive manner.  
+
+## Members
+* Poojith Obeysekara (Club Lead) [@LinkedIn]( https://www.linkedin.com/in/poojithobeysekara/)
```
Comment (dhanushkac) [inline cc/ICBT/2022-current.md]:
```suggestion
-  **Poojith Obeysekara (Club Lead)**
```

**Review by dhanushkac — APPROVED** [review]

**Pull request merged**